helpers/mathjax.js

```diff
@@ -129,12 +113,18 @@ export function loadMathJax(mathJaxConfig) {
 				const AbstractHandler = window.MathJax._.core.Handler.AbstractHandler.prototype;
 				const startup = window.MathJax.startup;
 
+				const getFirstChild = doc => {
+					const child = doc.firstChild;
+					if (!child || child.nodeType === Node.ELEMENT_NODE) return child;
+					else return child.nextElementSibling;
+				};
+
 				//
 				//  Extend HTMLAdaptor to handle shadowDOM as the document
 				//
 				class ShadowAdaptor extends HTMLAdaptor {
```

Comment:
This implementation does keep our custom adaptor. Brief forays into removing it have so far proven fruitless, so I'm not optimistic we can ditch it. MathJax just stores too much document-level state. However, I will keep poking at it to see if there's a path forward where we can just move to their default APIs.